request-type.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Request } from 'express'
+import { User } from '/user/user.model'
+
+interface RequestdWithUser extends {user: User} {
```

Comment:
```suggestion
interface RequestWithUser extends {user: User} {
```

Reply:
it extends {user: User} but in it's declaration there's
` user: user; ` . what's that user and why the field is overwritten?